### ISSUE TITLE
Add callbacks to scheduler to know when queue has been flushed/cleared

### ISSUE THREAD

**Issue by berendkleinhaneveld** [Korijn/observ#127]
While playing around with Kolla (collagraph playground with fine-grained reactivity instead of fibers/VDOM) I bumped into a problem: in the collagraph api, the render call takes a `callback` argument that gets called whenever a render of the VDOM has been performed. But in Kolla, there is no such thing as 'rendering of the VDOM' and any reactivity is all just based on a bunch of watchers.

When working with Pygfx, it is important to know when to request a draw call, and with current collagraph it was just as simple as requesting a draw at the end of a VDOM render. But in Kolla's case, I think what I need is a way to know when the scheduler in observ has flushed and cleared its queue! This could be the signal to Kolla to indicate that the state has been updated and that now the canvas should redrawn (or rather the callback should be called). In order to really know whether a draw call is needed, I could hook into the renderer, to see if any of its methods have been called since the last flush/draw call.

TL;DR: are you ok with me adding a way to add handlers to the scheduler in order to know when the queue has been flushed?

**Comment by berendkleinhaneveld** [comment]
Will solve in another way!
